node_modules/col-whiteboards/lib/api.js

```diff
@@ -2001,6 +2008,7 @@ var scheduleThumbnails = function() {
     Collabosphere.updateLastWhiteboardThumbnails();
 
     // Schedule the next run
+    log.info({'course': course}, 'Will run whiteboard thumbnail generator in %d seconds', THUMBNAIL_INTERVAL);
```

Comment:
@johncrossman There's no specific `whiteboard.id` here - the generator runs on all whiteboards that have been scheduled.